### ISSUE TITLE
Devices import dont work anymore (In Setup)

### ISSUE THREAD

**Issue by netzint** [linuxmuster/linuxmuster-base7#58]

**Comment by HappyBasher** [comment]
Cannot reproduce this. Logfile?

**Comment by zefanja** [comment]
What's in the logfile? (`/var/log/linuxmuster/setup/` AFAIK)

**Comment by HappyBasher** [comment]
Yes, logfiles would help.

**Comment by PLanB2008** [comment]
[setupLog.zip](https://github.com/linuxmuster/linuxmuster-base7/files/2772654/setupLog.zip)
Device import problem occured again. All logfies provided

**Comment by jeffbeck** [comment]
devices.csv??

**Comment by PLanB2008** [comment]
![grafik](https://user-images.githubusercontent.com/26460239/51383956-2fb57980-1b1b-11e9-9227-83eb4c34e833.png)

**Comment by PLanB2008** [comment]
output sophomorix-device --sync
seems to stop at this point:

![grafik](https://user-images.githubusercontent.com/26460239/51384168-c6823600-1b1b-11e9-8a38-1e1217df4954.png)

**Comment by PLanB2008** [comment]
So and why is it necessary to have a internet connection? 

Something during the setup process goes wrong. First there is a internet connection provided by opensense. Once This configuration is done the connection is not available anymore.

So in fact we run in two different problems.

**Comment by PLanB2008** [comment]
result of sophomorix-device --sync

[sync.txt](https://github.com/linuxmuster/linuxmuster-base7/files/2772793/sync.txt)

**Comment by jeffbeck** [comment]
The problem is the command:
/usr/bin/smbclient -U administrator%'yezNQ1nbd5Fnf' //linuxmuster.local/abc -c 'mkdir "program"'

The command pauses a long time when there is no internet connection available (Tested with a paused firewall in virtualbox)

Using the IP instead of linuxmuster.local cures the problem. So it must be a DNS issue.
host linuxmuster.cocal fail without internet connection:
`;; connection timed out; no servers could be reached`

The strange thing is, that it sometimes failes, sometimes not.

**Comment by HappyBasher** [comment]
Has opnsense been upgraded to the current version 18.7? This is essential.

**Comment by HappyBasher** [comment]
Success! The firewall's host alias definition was somewhat broken so the the privileged ips were no longer privileged. Fixed in 7.0.30.

**Comment by PLanB2008** [comment]
> The problem is the command:
> /usr/bin/smbclient -U administrator%'yezNQ1nbd5Fnf' //linuxmuster.local/abc -c 'mkdir "program"'
> 
> The command pauses a long time when there is no internet connection available (Tested with a paused firewall in virtualbox)
> 
> Using the IP instead of linuxmuster.local cures the problem. So it must be a DNS issue.
> host linuxmuster.cocal fail without internet connection:
> `;; connection timed out; no servers could be reached`
> 
> The strange thing is, that it sometimes failes, sometimes not.

I would like to reopen the issue. The setup of the lmn appliance should not need a running internet connection, we've to find a way to work around this.

**Comment by PLanB2008** [comment]
I somehow nailed it down, the server somehow needs a connection to the internet. We shoud definetely fix this...
This is really annoying in an environment which is not a default setup. In my current environment I had to add a nameserver and gateway manually which got overwritten by the setup.
To temporally solve the problem I had to add it again manually during the setup. 

Can we move this issue to sophomorix?
